### ISSUE TITLE
Lack of activity in the repository for Polish translation

### ISSUE THREAD

**Issue by kplich** [javascript-tutorial/en.javascript.info#2513]
Hello! Unfortunately, there seems to be zero activity in the repository since at least a few months: my pull request has been waiting for review since two weeks; the oldest open PRs date back to August 2019. I'd love to contribute more translations, but I don't see the point of doing so if the contributions won't make it to the repository/application.

**Comment by iliakan** [comment]
You can take it over then =)

**Comment by kplich** [comment]
Haha, that's the answer I was afraid of! ;) In theory, it sounds nice, but at the moment I don't think I have enough time to actively maintain the project. Thanks for a quick reply, though.

**Comment by Onxi95** [comment]
Unfortunately the problem still exists, current translations are not approved, 1470 commits are waiting to be synced with the upstream.

**Comment by iliakan** [comment]
The Polish translation needs a maintainer, knowledgeable enough to commit and approve.

**Comment by Onxi95** [comment]
Is a specific number of ready-to-go hours required? I would like to try to help, but unfortunately I have a tight schedule and I could spend 1-3 hours a week on it.

**Comment by iliakan** [comment]
Only @Onxi95 ? Anyone else?

**Comment by kdshop** [comment]
I can help too!

**Comment by kdshop** [comment]
I can help too!

**Comment by iliakan** [comment]
Polish? Did you ping the maintainer? @kdshop , @RafalMichniuk want to go maintainers?

**Comment by RafalMichniuk** [comment]
@iliakan not yet, but it will be next thing what I'm going to do :) 
Maintainer? Why not?! :D Let's give a try for it!

**Comment by RafalMichniuk** [comment]
@iliakan still no progress about it. I wrote an email to @kdshop but no response so far. Is there anything else we could do?

**Comment by iliakan** [comment]
I can make you a maintainer if you have enough JS experience to translate/review PRs.

**Comment by RafalMichniuk** [comment]
I guess I have enough experience but not sure about time. 1-2 hour per week is ok?

**Comment by iliakan** [comment]
Welcome to the team @RafalMichniuk ;)

**Comment by kdshop** [comment]
> Polish? Did you ping the maintainer? @kdshop , @RafalMichniuk want to go maintainers?

Yes sure I will also connect with @RafalMichniuk to set up some plan for works

**Comment by RafalMichniuk** [comment]
Hi @kdshop, how do you want to connect?
